### PR TITLE
Add note with link to k8s compatibility info

### PR DESCRIPTION
**Pull request by mattlord**
Signed-off-by: Matt Lord <mattalord@gmail.com>

### CODE REVIEW

File: content/en/docs/13.0/get-started/operator.md [hunk 1]
```diff
@@ -5,10 +5,12 @@ featured: true
 aliases: ['/docs/tutorials/kubernetes/','/user-guide/sharding-kubernetes.html', '/docs/get-started/scaleway/','/docs/get-started/kubernetes/']
 ---
 
-PlanetScale provides a [Vitess Operator for Kubernetes](https://github.com/planetscale/vitess-operator), released under the Apache 2.0 license. The following steps show how to get started using Minikube:
+PlanetScale provides a [Vitess Operator for Kubernetes](https://github.com/planetscale/vitess-operator), released under the Apache 2.0 license.. The following steps show how to get started using Minikube:
```
Comment (deepthi) [inline content/en/docs/13.0/get-started/operator.md]: looks like a typo?

**Review by deepthi — COMMENTED** [review]
It looks like you overwrote the 11.0 and 12.0 versions of this doc with the latest instead of just adding a line/link to the supported versions information.
Is there a reason for that?

File: content/en/docs/13.0/get-started/operator.md (hunk 1 above)
Comment (mattlord) [reply content/en/docs/13.0/get-started/operator.md]: Good catch! Fixed

**Comment by mattlord** [comment]
> It looks like you overwrote the 11.0 and 12.0 versions of this doc with the latest instead of just adding a line/link to the supported versions information.
> Is there a reason for that?

Somehow [my changes here](https://github.com/vitessio/website/pull/904) — done before the versioned docs — only got applied to 13.0. There's nothing specific to vitess versions in this doc AFAIK.

**Comment by deepthi** [comment]
> > It looks like you overwrote the 11.0 and 12.0 versions of this doc with the latest instead of just adding a line/link to the supported versions information.
> > Is there a reason for that?
> 
> Somehow [my changes here](https://github.com/vitessio/website/pull/904) — done before the versioned docs — only got applied to 13.0. There's nothing specific to vitess versions in this doc AFAIK.

v11.0 and v12.0 of docs were created from the state of the website repo on (approximately) the GA release dates for those releases. So any changes made to specific docs after those dates won't be in v11.0 / v12.0 of the docs.
I think you will need to revert the changes at least to the 11.0 version of this doc. The operator version for that Vitess release was v2.4 which is documented to work with 1.15-1.17, not 1.17-1.19.

**Comment by mattlord** [comment]
> > > It looks like you overwrote the 11.0 and 12.0 versions of this doc with the latest instead of just adding a line/link to the supported versions information.
> > > Is there a reason for that?
> > 
> > 
> > Somehow [my changes here](https://github.com/vitessio/website/pull/904) — done before the versioned docs — only got applied to 13.0. There's nothing specific to vitess versions in this doc AFAIK.
> 
> v11.0 and v12.0 of docs were created from the state of the website repo on (approximately) the GA release dates for those releases. So any changes made to specific docs after those dates won't be in v11.0 / v12.0 of the docs. I think you will need to revert the changes at least to the 11.0 version of this doc. The operator version for that Vitess release was v2.4 which is documented to work with 1.15-1.17, not 1.17-1.19.

OK, I reverted the 11.0 changes.

**Review by hallaroo — APPROVED** [review]
LGTM

**Pull request merged**